bigbluebutton-html5/private/locales/en.json

```diff
@@ -25,8 +25,8 @@
   "app.userlist.menu.kickUser.label": "Kick user",
   "app.userlist.menu.muteUserAudio.label": "Mute user",
   "app.userlist.menu.unmuteUserAudio.label": "Unmute user",
-  "app.userlist.useritem.nostatus.arialabel": "{username} {presenter} {you}",
-  "app.userlist.useritem.status.arialabel": "{username} {presenter} {you} current status {status}",
+  "app.userlist.useritem.nostatus.arialabel": "{0} {1} {2}",
```

Comment:
@oswaldoacauan 
The translation will be exactly the same - "{0} {1} {2}"
It's a small correction after this PR: https://github.com/bigbluebutton/bigbluebutton/pull/3951
Switching dynamic variables' names from strings to numbers, otherwise translators can translate them by mistake, which will break localization.